test_security/test/test_invalid_secure_node_creation_c.cpp

```diff
@@ -84,14 +84,15 @@ class CLASSNAME (TestInvalidSecureNode, RMW_IMPLEMENTATION) : public ::testing::
     rcl_ret_t ret = rcl_node_fini(this->node_ptr);
     delete this->node_ptr;
     ret = rcl_shutdown(&context);
-    EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+    (void)ret;
```

Comment:
@ivanpauno why removing the check on `ret`? Also, this is an odd (preexisting) fixture. I would've expected `SetUp()` to initialize both node and context, or otherwise have each test taking care of proper shutdown.

Reply:
> why removing the check on ret?

Depending on the rmw implementation, some of the test case might fail when shutdown.

> I would've expected SetUp() to initialize both node and context, or otherwise have each test taking care of proper shutdown.

I will do the second.

Reply:
See https://github.com/ros2/system_tests/pull/406/commits/dd9e967163d648ca31499081b5e1294a7f1c55a6.